conf/modules.config

```diff
@@ -447,6 +497,22 @@ process {
     }
 
     withName: HAMRONIZATION_SUMMARIZE {
+        publishDir = [
+            path: { "${params.outdir}/reports/hamronization_summarize" },
+            mode: params.publish_dir_mode,
+            saveAs: { (params.run_taxonomic_classification == false) ? it : null }
+        ]
+    }
+
+    withName: MERGE_TAXONOMY_HAMRONIZATION {
+        publishDir = [
+            path: { "${params.outdir}/reports/hamronization_summarize" },
+            mode: params.publish_dir_mode,
+            saveAs: { _ -> null } //do not save the file
```

Comment:
Is that  because is present in finasl merged table?

Reply:
yup, so this generates an intermediate summary file that is not required to be published as the info. it contains is in the final merged table